ehcache-impl/src/main/java/org/ehcache/impl/serialization/CompactJavaSerializer.java

```diff
@@ -80,8 +85,12 @@ public static <T> Class<? extends Serializer<T>> asTypedSerializer() {
 
   @Override
   public void init(final StateRepository stateRepository) {
-    this.persistentState = stateRepository.getPersistentStateHolder("CompactJavaSerializer-ObjectStreamClassIndex", Integer.class, ObjectStreamClass.class, c -> true, null);
-    refreshMappingsFromStateRepository();
+    try {
+      this.persistentState = stateRepository.getPersistentStateHolder("CompactJavaSerializer-ObjectStreamClassIndex", Integer.class, ObjectStreamClass.class, c -> true, null);
+      refreshMappingsFromStateRepository();
+    }catch (CachePersistenceException e){
```

Comment:
Not sure I understand this change here, the init throws exception and we eat it up without letting user know the init has failed. Care to elaborate?

Reply:
Explanation same as above comment. We will be reverting this change.